commands-yml/commands/element/text.yml

```diff
@@ -0,0 +1,78 @@
+---
+name: Get Element Text
+short_description: Returns visible text for element
+description:
+  >
+     Returns the visible text for the element.
+
+example_usage:
+  java:
+    |
+      MobileElement element = (MobileElement) driver.findElementByClassName("SomeClassName");
+      let elText = element.getText();
+  python:
+    |
+      # el = self.driver.find_element_by_accessibility_id('SomeId')
+      # TODO: Python example
+  javascript_wd:
+    |
+      let element = await driver.elementByAccessibilityId("SomeAccessibilityID");
+      await element.submit();
+  javascript_wdio:
+    |
+      driver.getText("~SomeAccessibilityId");
+  ruby:
+    |
+      # TODO Ruby sample
+  php:
+    |
+      // TODO PHP sample
+  csharp:
+    |
+      // TODO C# sample
+
+client_docs:
+  java: "https://seleniumhq.github.io/selenium/docs/api/java/org/openqa/selenium/WebElement.html#getText--"
+  python: "http://selenium-python.readthedocs.io/api.html" # TODO Python documentation link
+  javascript_wdio: "http://webdriver.io/api/property/getText.html"
+  javascript_wd: "https://github.com/admc/wd/blob/master/lib/commands.js#L1832"
+  ruby: "http://www.rubydoc.info/gems/selenium-webdriver/Selenium/WebDriver/" # TODO Ruby documentation link
+  php: "https://github.com/appium/php-client/" # TODO PHP documentation link
+  csharp: "https://github.com/appium/appium-dotnet-driver/" # TODO Dotnet documentation link
+
+# Driver support by platform
+driver_support:
+  ios:
+    xcuitest: true
+    uiautomation: true
+  android:
+    uiautomator2: true
+    uiautomator: true
+  mac:
+    mac: true
+  windows:
+    windows: true
+client_support:
+  java: true
+  python: true
+  ruby: true
+  php: true
+  csharp: true
+  javascript_wd: true
+  javascript_wdio: true
+  
+
+# Information about the HTTP endpoints
+endpoint:
+  url: /wd/hub/session/:session_id/element/:element_id/text
+  method: 'GET'
+  url_parameters:
+    - name: session_id
+      description: ID of the session to route the command to
+    - name: element_id
+      description: ID of the element to route the command to
```

Comment:
i wouldn't say this command is being routed to an element. i'd just say it's the id of the element whose text we want.